File: app/src/main/java/com/quibbler/sevenmusic/listener/BroadcastMusicPlayListener.java
package com.quibbler.sevenmusic.listener;

/**
 * Package:        com.quibbler.sevenmusic.listener
 * ClassName:      BroadcastMusicPlayListener
 * Description:    接收广播，处理具体事务
 * Author:         zhaoopeng
 * CreateDate:     2019/9/27 18:41
 */
public interface BroadcastMusicPlayListener extends BroadcastListener {
    public void handBroadcast();

    public void onMusicPlay();

    public void onMusicPause();
}

File: app/src/main/java/com/quibbler/sevenmusic/listener/BroadcastMusicPlayListener.kt
package com.quibbler.sevenmusic.listener

/**
 * Package:        com.quibbler.sevenmusic.listener
 * ClassName:      BroadcastMusicPlayListener
 * Description:    接收广播，处理具体事务
 * Author:         zhaoopeng
 * CreateDate:     2019/9/27 18:41
 */
interface BroadcastMusicPlayListener : BroadcastListener {
    fun handBroadcast()

    fun onMusicPlay()

    fun onMusicPause()
}
